alibi/saving.py

```diff
@@ -373,6 +376,14 @@ def _load_CounterfactualRLTabular(path: Union[str, os.PathLike],
     return _helper_load_CounterfactualRL(path, predictor, explainer)
 
 
+def _save_ProtoSelect(path: Union[str, os.PathLike]) -> None:
+    raise NotImplementedError('ProtoSelect saving functionality not implemented.')
```

Comment:
Would be good to expand on the pros/cons for saving the kernel matrix etc.

Reply:
The only pros will be that we can store some hyperparameters from cross-validation. The bad thing is that is quadratic in the number of elements and storing the matrix can be a bit demanding ... an alternative will be to store only the params but not the kernel matrices. This would mean that it requires refitting and I guess this is connected with another comment related to the API. Better discuss it offline ...

Reply:
Decided to not implement saving/loading functionality because the data is "fixed" in the sense that you can't reuse the same object for finding prototypes in new data (unlike explainers, ML models etc.)